### PR TITLE
messagebird support (sms only)

### PR DESCRIPTION
**Pull request by wleese**
SMS only messagebird support.

https://github.com/linkedin/iris/issues/364

### CODE REVIEW

File: src/iris/vendors/iris_messagebird.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        start = time.time()
+        payload = self.get_message_payload(message)
+
+        if self.debug:
+            logger.info('debug: %s', payload)
+        else:
+            try:
+                response = requests.post(self.endpoint_url,
+                                         headers=self.headers,
+                                         data=json.dumps(payload),
```
Comment (jrgp) [inline src/iris/vendors/iris_messagebird.py]: You could instead do:

    json=payload

And requests will encode to json for you

File: src/iris/vendors/iris_messagebird.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                                         proxies=self.proxy)
+                if response.status_code == 201:
+                    return time.time() - start
+                else:
+                    logger.error('Failed to send message to messagebird: %d',
+                                 response.status_code)
```
Comment (jrgp) [inline src/iris/vendors/iris_messagebird.py]: Can you please consolidate these two loggers into a single one ?

**Comment by jrgp** [comment]
Cool, looks like you need to rip out the `import json` at the top and then it'll be good.

**Comment by jrgp** [comment]
Coolio. Thanks for the patch! Now we have some variety in case we want to move off of twilio

**Comment by wleese** [comment]
No problem - thanks for the patience.

**Pull request merged**